src/main.c

```diff
@@ -23,7 +23,19 @@
 #include <glib.h>
 #include <gio/gio.h>
 
-#include "device.h"
+#include "../include/device.h"
+
+#ifdef UNUSED
```

Comment:
Yet another implementation for UNUSED? glib already has one for GNUC: https://developer.gnome.org/glib/stable/glib-Miscellaneous-Macros.html#G-GNUC-UNUSED:CAPS